Assets/Scripts/Localization/LocalizationDownloader.cs

```diff
@@ -226,9 +226,9 @@ private static void OnDownloadLocalizationComplete(Action onLocalizationDownload
                 // called ProjectPorcupineLocalization-*branch name*. Now we need to move all files from that directory
                 // to Application.streamingAssetsPath/Localization.
                 FileInfo[] fileInfo = localizationFolderInfo.GetFiles();
-                if (fileInfo.Length > 0)
+                if (fileInfo.Length > 2)
```

Comment:
I really hate lambda expressions - they are really unclear to me and every time I see one I need to spend like 5 minutes figuring out what it does - hence I didn't use it.